stories/Inputs/Slider.stories.js

```diff
@@ -1,18 +1,91 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import { withKnobs, text, boolean, number } from '@storybook/addon-knobs';
+import { withKnobs, boolean } from '@storybook/addon-knobs';
 import { Slider } from '../../src';
+import { mockElement } from '../Tools';
 
 storiesOf('Slider', module)
     .addDecorator(withKnobs)
-    .add('with default', () => (
-        <Slider
-            max={number('max', 10)}
-            min={number('min', 0)}
-            step={number('step', 1)}
-            disabled={boolean('disabled', false)}
-            value={number('value', 3)}
-            trackColor={text('trackColor', '#010101')}
-            buttonColor={text('buttonColor', 'White')}
-        />
-    ));
+    .add(
+        'basic',
+        mockElement(
+            ([state, setState]) => (
+                <Slider
+                    min={1}
+                    max={100}
+                    onChange={target => {
+                        setState(target);
+                    }}
+                    values={state}
+                ></Slider>
+            ),
+            '',
+        ),
+    )
+    .add(
+        'with rail',
+        mockElement(
+            ([state, setState]) => (
+                <Slider
+                    min={1}
+                    max={100}
+                    hasRail={boolean('hasRail', true)}
+                    onChange={target => {
+                        setState(target);
+                    }}
+                    values={state}
+                ></Slider>
+            ),
+            '',
+        ),
+    )
+    .add('disabled', () => (
+        <Slider disabled={boolean('disabled', true)}></Slider>
+    ))
+    .add(
+        'with popup',
+        mockElement(
+            ([state, setState]) => (
+                <Slider
+                    min={20}
+                    max={50}
+                    padding="50px"
+                    hasPopup={boolean('hasPopup', true)}
+                    onChange={target => {
+                        console.log(target);
```

Comment:
these are stories but perhaps this log may not be needed in this version here as there is a setState